core/src/main/java/org/libreoffice/lots/db/QueryResultsList.java

```diff
@@ -67,11 +66,11 @@ public QueryResultsList(List<? extends Dataset> datasets)
   public int size() { return data.size();}
 
   @SuppressWarnings("unchecked")
-  // Die Typsicherheit kann hier nicht gefährdet sein, da laut
-  // http://docs.oracle.com/javase/tutorial/java/generics/wildcardGuidelines.htm vor
-  // allem gesichert sein muss, dass kein falscher Typ zur List<? extends Dataset>
-  // hinzugefügt wird. So etwas ist mit einem Iterator, der nur die Methoden
-  // hasNext(), next() und remove() kennt, nicht möglich.
+  // The type safety cannot be endangered here because it is loud
```

Comment:
Do you understand the German or is this an automatic translation? This is wrong.

Reply:
No! I was using ChatGPT and Google Translate, I translated each comment using both the tools and the output were same so, I thought is must be correct.

Reply:
Please try translating this block with DeepL:
https://www.deepl.com/translator#de/en/Die%20Typsicherheit%20kann%20hier%20nicht%20gef%C3%A4hrdet%20sein%2C%20da%20laut%20http%3A%5C%2F%5C%2Fdocs.oracle.com%5C%2Fjavase%5C%2Ftutorial%5C%2Fjava%5C%2Fgenerics%5C%2FwildcardGuidelines.htm%20vor%20allem%20gesichert%20sein%20muss%2C%20dass%20kein%20falscher%20Typ%20zur%20List%3C%3F%20extends%20Dataset%3E%20hinzugef%C3%BCgt%20wird.%20So%20etwas%20ist%20mit%20einem%20Iterator%2C%20der%20nur%20die%20Methoden%20hasNext()%2C%20next()%20und%20remove()%20kennt%2C%20nicht%20m%C3%B6glich.